openlibrary/templates/home/stats.html

```diff
@@ -15,13 +15,13 @@ <h2 class="inline">$_('Around the Library')</h2>
         </div>
 
         <div class="statschart">
-          <div class="chartShow" onClick="window.location.href='/recentchanges';" title="$_('How many new Open Library members have we welcomed?')">
+          <div class="chartShow" onClick="window.location.href='//archive.org/stats';" title="$_('How many new Open Library members have we welcomed?')">
```

Comment:
I believe people on the issue were referring to the openlibrary stats page; not the archive.org stats page. The stats on archive.org/stats are more for archive.org :)

```suggestion
          <div class="chartShow" onClick="window.location.href='//openlibrary.org/stats';" title="$_('How many new Open Library members have we welcomed?')">
```